example_scenes/customtex.py

```diff
@@ -31,8 +31,8 @@ def construct(self):
         # Set the compiler and output format (default: latex and .dvi)
         # possible tex compilers: "latex", "pdflatex", "xelatex", "lualatex", "luatex"
         # possible output formats: ".dvi",  ".pdf", and ".xdv"
-        myTemplate.tex_compiler = "pdflatex"
-        myTemplate.output_format = ".pdf"
+        myTemplate.tex_compiler = "latex"
+        myTemplate.output_format = ".dvi"
```

Comment:
Please revert as well. This doesn't fit within the scope of this PR as it's not a grammatical error -- it's code.

Reply:
Done